utilities/policy_utilities.mak

```diff
@@ -28,8 +28,11 @@ TARGET_MACHINE_TYPE= x64
 endif
 
 S= $(SRC_DIR)
-CERT_SRC=$(CERTIFIER_PROTOTYPE_DIR)/src
 O= $(OBJ_DIR)
+I= $(INC_DIR)
```

Comment:
Previously `I` and `US` (one line below) were not defined, but were being used in the make commands.

(Surprisingly, things somehow worked ok ...) 

I defined these here based on how they were being defined in the companion `cert_utility.mak` file.